src/api/Feed.js

```diff
@@ -678,21 +673,40 @@ class Feed extends Base {
     /**
      * Network error callback
      *
+     * @param {boolean} hasError - true if the UI should display an error
      * @param {Error} error - Axios error object
+     * @param {Function} errorCallback - the optional error callback to be executed
+     * @param {string} code - the error code for the error which occured
      * @return {void}
      */
-    errorCallback = (error: $AxiosXHR<any>): void => {
-        console.error(error); // eslint-disable-line no-console
-        this.hasError = true;
+    errorCallback = (
+        hasError: boolean = false,
+        e: ElementsXhrError,
+        errorCallback?: ErrorCallback,
+        code?: string,
+    ): void => {
+        if (hasError) {
```

Comment:
It determines if the UI should show an error or not. This will be false in the case it is not a user correctable error, but should still call the error callback